float64 elapsedTime
float64 x
float64 y
float64 th
float64 linear
float64 angular
float64 error
float64 deltaError
float64 leftSide
float64 leftForward
float64 rightForward
float64 rightSide
bool isServoOn
bool isTurnRight
bool isWallFront
